Assets/MixedRealityToolkit/UX/Scripts/Tooltip/ToolTipConnector.cs

```diff
@@ -0,0 +1,258 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using UnityEngine;
+
+namespace MixedRealityToolkit.UX.ToolTips
+{
+    /// <summary>
+    /// Connects a ToolTip to a target
+    /// Maintains that connection even if the target moves
+    /// </summary>
+    public class ToolTipConnector : MonoBehaviour
+    {
+        public enum FollowTypeEnum
```

Comment:
Can we separate out these Enums and if possible make them more generic? (one for each enum).  Also don't suffix them with Enum, as this is determined by its type.
Seems these are fairly common UX style enums that should be standardised and reused.

If these really are ONLY for this component, then they should be private.